docs/contributing/index.md

```diff
@@ -192,3 +192,45 @@ Once your changes are ready, click on the "Commit changes..." button in the top
 
     * if the change is small and you are CSCS staff, you can merge the PR immediately
     * all other changes can be
+
+## Style Guide
+
+This section contains general guidelines for how to format and present documentation in this repository.
+They should be followed for most cases, but as a guideline it can be broken, _with good reason_.
+
+### Headings are Written in Title Case
+
+Use [title case](https://en.wikipedia.org/wiki/Letter_case#Title_case) for headings, meaning all words are capitalized except for minor words.
+
+### Avoid Nesting Headings too Deep
+
+Nesting headings up to three levels is generally ok.
+
+### Unordered and Ordered Lists
+
+Write lists as proper sentences.
+Separate the items simply with commas if each item is simple, or make each item a full sentence if the items are longer and contain multiple sentences.
+
+1. The first item can look like this,
+2. the second like this, and
+3. the third item like this.
+
+### Using Admonitions
+
+Aim to include examples, notes, warnings using [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) whenever appropriate.
+They stand out better from the main text, and can be collapsed by default if needed.
+
+!!! example "Example one"
```

Comment:
Which style should be used for the title?

Reply:
The text in the example says sentence case, because it's also my preference here.

Reply:
Sorry, I missed it was part of the example.